services/movie.py

```diff
@@ -0,0 +1,47 @@
+from db.models import Movie, Genre, Actor
+from django.db.models import QuerySet
+
+
+def get_movies(
+    genres_ids: list[int] = None,
+    actors_ids: list[int] = None,
+) -> QuerySet | Movie:

```

Comment:
The return type annotation `QuerySet | Movie` is incorrect. Since the function always returns a `QuerySet`, it should be `QuerySet` only. Update the return type annotation to reflect this.